### PR TITLE
Fix typos using aspell

### PR DESCRIPTION
**Pull request by katrinleinweber**
While working on #459, I noticed a few typos and ran

```bash
for f in **/*.md
	do aspell check $f
done
```

just to be sure.

### CODE REVIEW

File: topic_folders/governance/lesson-program-policy.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ This Policy outlines the Roles and Responsibilities of the Lesson Programs and T
 - The Carpentries will include the Lesson Programs in assessment efforts, including survey development and data analysis. 
 - The Carpentries will ensure the instructor training program addresses the training needs of the Lesson Program, incorporating examples and exercises targeted towards the communities served by the Lesson Program.
 - The Carpentries will prepare Trainers to work with trainees from the Lesson Program. 
-- The Carpentries Instructor Training recruitment and retention strategies will include building communities of InstructorsIinstructors in communities and domains served by the Lesson Program. 
+- The Carpentries Instructor Training recruitment and retention strategies will include building communities of Instructors in domains served by the Lesson Program.
```
Comment (katrinleinweber) [inline topic_folders/governance/lesson-program-policy.md]: Please esp. review this change. My suggestion is not the only valid resolution here.

Comment (katrinleinweber) [inline topic_folders/governance/lesson-program-policy.md]: Please esp. review this change. My suggestion is not the only valid resolution here.

**Comment by maneesha** [comment]
This is wonderful @katrinleinweber! Thank you so much.
(and I'm also excited I got to learn about `aspell` through this!)

File: topic_folders/governance/lesson-program-policy.md (hunk 1 above)
Comment (maneesha) [reply topic_folders/governance/lesson-program-policy.md]: I put the word "communities" back in - otherwise this is great.

**Pull request merged**